src/ir_Daikin.cpp

```diff
@@ -2310,7 +2312,7 @@ void IRDaikin176::setFan(const uint8_t fan) {
   else if (fan < kDaikinFanMin || fan > kDaikinFanMax)
```

Comment:
`kDaikinFanMax` is `5`

Your comment says it's 3. Thus you can't use that same constant. You need a new one, if your comment is correct. e.g. 3.

Try:
`const uint8_t kDaikin176FanMax = 3;`